client/ayon_core/hosts/nuke/plugins/create/create_write_prerender.py

```diff
@@ -46,11 +46,15 @@ def get_pre_create_attr_defs(self):
         return attr_defs
 
     def create_instance_node(self, product_name, instance_data):
+        settings = self.project_settings["nuke"]["create"]
+        settings = settings["CreateWritePrerender"]["instance_attributes"]
+
         # add fpath_template
         write_data = {
             "creator": self.__class__.__name__,
             "productName": product_name,
-            "fpath_template": self.temp_rendering_path_template
+            "fpath_template": self.temp_rendering_path_template,
+            "headless_farm_submission": "headless_farm_submission" in settings
```

Comment:
Should this be
```suggestion
            "headless_farm_submission": settings.get("headless_farm_submission", False)
```

Reply:
No, the `instance_attributes` is a list, not dict.